tests/fixture/testnet/config.go

```diff
@@ -30,9 +35,18 @@ import (
 const (
 	DefaultNodeCount      = 5
 	DefaultFundedKeyCount = 50
+
+	DefaultGasLimit = uint64(10_0000_000) // Arbitrary gas limit is arbitrary
+
+	// Arbitrarily large amount of AVAX to fund keys on the X-Chain for testing
+	DefaultFundedKeyXChainAmount = 30 * units.MegaAvax
 )
 
 var (
+	// The actual value needs to be set in init() because it's not
+	// possible to statically define a value large enough for C-Chain.
+	DefaultFundedKeyCChainAmount big.Int
```

Comment:
```suggestion
	// Arbitrarily large amount of AVAX (10^12) to fund keys on the C-Chain for testing
	DefaultFundedKeyCChainAmount = new(big.Int).Exp(big.NewInt(10), big.NewInt(30), nil)
```
This is 10^12 AVAX because the units on the C-chain is 10^-18 AVAX.